openmetadata-spec/src/main/resources/json/schema/entity/services/connections/database/deltaLakeConnection.json

```diff
@@ -11,6 +11,39 @@
       "type": "string",
       "enum": ["DeltaLake"],
       "default": "DeltaLake"
+    },
+    "metastoreHostPortConnection": {
+      "title": "Hive Metastore Service",
```

Comment:
we need to pass `"type":"object"` here otherwise it causes some parsing issue in UI, have face similar issue for oracledb